source/psychlua/FunkinLua.hx

```diff
@@ -982,6 +983,18 @@ class FunkinLua {
 			LuaUtils.loadFrames(leSprite, image, spriteType);
 			game.modchartSprites.set(tag, leSprite);
 		});
+		Lua_helper.add_callback(lua, "makeParallaxSprite", function(tag:String, ?image:String = null, ?x:Float = 0, ?y:Float = 0) {
+			tag = tag.replace('.', '');
+			LuaUtils.resetSpriteTag(tag, true);
+			var leSprite:ParallaxSprite = new ParallaxSprite(x, y, Paths.image(image));
```

Comment:
@CharlesCatYT FunkinLua references the ParallaxSprites for the lua callbacks the same way ModchartSprites are, so the import is needed!